camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeper.java

```diff
@@ -222,8 +226,24 @@ public void run() throws Exception
       {
         System.err.println("unable to process " + topicName + " skipping...");
         e.printStackTrace();
-        executorService.shutdown();
-        throw e;
```

Comment:
what if we caught and ignored a bad exception here?  what will be the `exit status`? `0=success`?